contracts/core/ERC5018.sol

```diff
@@ -26,6 +29,14 @@ contract ERC5018 is LargeStorageManager, BlobStorageManager, IERC5018, ISemver {
         return storageModes[keccak256(name)];
     }
 
+    function setStorageContract(address storageAddress) public onlyOwner {
```

Comment:
Why do we move these functions to the child contract?

Reply:
`BlobStorageManager` and `LargeStorageManager` should focus solely on business logic, and the `Ownable` contract is better placed within `ERC5018`. The corresponding onlyOwner functions have also been moved to ERC5018.

Reply:
I think storageContract is only visible to BlobStorageManager, and ERC5018 should not know it

Reply:
revoked